UNRELEASED_TOKENS.md

```diff
@@ -0,0 +1,52 @@
+# Guide
+
+Read [#2093](https://github.com/hoprnet/hoprnet/issues/2093) for context.
+
+## Words
+
+| word            | description                                                                |
+| --------------- | -------------------------------------------------------------------------- |
+| HoprDistributor | a live smart contract on **mainnet** that users can claim HOPR tokens      |
+| CLAIMANT_ADDR   | an **ethereum address** that can claim HOPR tokens through HoprDistributor |
+| HOPR_ID         | a **HOPR id** of a HOPR node                                               |
+
+## Process
+
+1. User sends an email to tech@hoprnet.org with subject `Link my HOPR node`, and a message containing their `HOPR_ID` and the signature of `Link my Ethereum address to HOPR node address <HOPR_ID>` that was signed using `CLAIMANT_ADDR`. Example:
+
+```
+16Uiu2HAmQBZA4TzjKjU5fpCSprGuM2y8mpepNwMS6ZKFATiKg68h
+<signature of "Link my Ethereum address to HOPR node address 16Uiu2HAmQBZA4TzjKjU5fpCSprGuM2y8mpepNwMS6ZKFATiKg68h>
+```
+
+2. PM creates an issue containing user's email and assigns a [team member](#who-is-team-member) to tackle.
+3. Team member must then verify the validity of the request:
+   1. Signer address (aka `CLAIMANT_ADDR`) of the signature has claimable tokens on schedules `EarlyTokenBuyers` or `TeamAndAdvisors` _(link to etherscan method)_.
+   2. Signer does not already exist in [unreleasedTokens.json](./packages/cover-traffic/unreleasedTokens.json)
+   3. Inserted `HOPR_ID` is a valid HOPR ID _(link to tooling)_
+4. Team member adds another entry to [unreleasedTokens.json](./packages/cover-traffic/unreleasedTokens.json) using syntax `{ tokens: string, ethAddress: string, hoprId: string }[]`
```

Comment:
What about adding a `blockNumber` or some on-chain proof to it?

Reply:
merging a PR requires approval, both author and reviewer need to check the data - not sure what kind of proof author can provide

Reply:
Specifying the `blockNumber` of the token `Transfer` event gives a clear argument how to check that the signer has indeed the requested tokens.

Afterwards, in case of a dispute, it is easy to prove or disprove whether the claimant possessed at the requested point in time the tokens and that the team member has done their job correctly.

Reply:
this is all in the blockchain, I don't think it's necessary, if there is a dispute we can always check